crates/types/src/config/admin.rs

```diff
@@ -64,6 +65,12 @@ pub struct AdminOptions {
     /// The default replication strategy to be used by the cluster controller to schedule partition
     /// processors.
     pub default_replication_strategy: ReplicationStrategy,
+
+    /// # Log Controller error retry policy
+    ///
+    /// The retry policy for log controller when it has
+    /// issue committing a reconfiguration
+    pub log_controller_error_retry_policy: RetryPolicy,
```

Comment:
I'd probably hard-code this for the time being, to avoid adding extra public config keys until we're sure we know what we want them to be :-) At least that's the approach we've taken in a few other places. I think the 15s default is sane for this purpose.